Sources/MongoSwift/ClientSession.swift

```diff
@@ -222,4 +222,72 @@ public final class ClientSession {
             }
         }
     }
+
+    /**
```

Comment:
Each of the new methods in this field could benefit from a `SeeAlso` link to the official MongoDB documentation on transactions (https://docs.mongodb.com/manual/core/transactions). See [here](https://github.com/mongodb/mongo-swift-driver/blob/master/Sources/MongoSwift/MongoCollection%2BChangeStreams.swift#L24) for an example of that.

Reply:
Done.